salt/orchestrate/edx/services/mongodb.sls

```diff
@@ -28,6 +33,8 @@ generate_mongodb_cloud_map_file:
           - {{ salt.boto_secgroup.get_group_id(
             'vault-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
         subnetids: {{ subnet_ids }}
+        profile_overrides:
+          size: t2.medium
```

Comment:
Why are we overriding it here with the same instance type that's already specified?